visualization/test/text_simple.cpp

```diff
@@ -0,0 +1,24 @@
+#include <pcl/point_types.h>
+
+#include <pcl/visualization/pcl_visualizer.h>
+
+int
+main (int argc, char** argv)
+{
+  pcl::visualization::PCLVisualizer viz ("Visualizator");
+  viz.addCoordinateSystem (1.0);
+
+  viz.addText3D ("Following text", pcl::PointXYZ(0.0, 0.0, 0.0),
+                 1.0, 1.0, 0.0, 0.0, "id_following");
+  viz.spin ();
```

Comment:
Isn't this a blocking call which continuously runs the main loop? Shouldn't `spinOnce` be used instead? This comment is applicable for all other call you do.

Reply:
In fact I was inspired by other tests from this directory, and the behavior is good. Blocking `spin()` method is correct. If you run the test, it allow you to see the effect of the different add/remove of text step by step. To pass to the next step, you just have to press "Q".

If I change it with `spinOnce()`, I will not be able to even see the window: it will open and close instantly.